frontend/package.json

```diff
@@ -43,8 +43,8 @@
     "watch-css": "yarn run build-css && node-sass-chokidar --include-path ./src src/ -o src/ --watch --recursive",
     "get-schema": "./src/stories/schema/getSchema.sh",
     "build": "react-scripts build",
```

Comment:
Wondering if the NODE_PATH=src matters for the non-local build. @briaguya, when did we think we should get a staging instance up to test things like this?

Reply:
Yea, I wasn't thrilled with this approach because it's not comprehensive and a more of a let's see what breaks and address it when it does. 
We can also do the approach of setting NODE_PATH in package.json for storybook and storyshots ONLY and .env files for all else. 
I went this route just because it's all in one place and compat with storybook.